server/services/party/actions.ts

```diff
@@ -1,13 +1,15 @@
 export class CreateParty {
     static readonly event = 'party.create'
 
-    constructor(public name: string, public characterName: string) {}
+    constructor(public partyName: string, public characterId: number) {
+    }
 }
 
 export class PartyCreated {
     static readonly event = 'party.created'
 
-    constructor(public world: string, public characterId: number) {}
+    constructor(public partyId:number, public characterId: number) {
```

Comment:
Events are scoped to the world so we should never need to provide a world parameter